src/client/java/minicraft/core/Renderer.java

```diff
@@ -185,6 +194,131 @@ public static void render() {
 		}
 	}
 
+	public static final AppStatusBar appStatusBar = new AppStatusBar();
+
+	public static class AppStatusBar {
```

Comment:
Perhaps move the AppStatusBar and it's implementations into it's own file?

And variables are usually declared at the top of the class.

Reply:
I am not so sure about this since it is only used in Renderer.